.github/workflows/tests.yml

```diff
@@ -76,7 +77,8 @@ jobs:
     skip_sonar:
         name: Skip SonarCloud in merge queue
         if: github.event_name == 'merge_group' || inputs.disable_coverage == 'true'
-        runs-on: ubuntu-latest
```

Comment:
This one only takes seconds, probably not worth moving to self hosted runners

Reply:
Yeah this was just to experiment with because it's quite simple.